testing/utils.py

```diff
@@ -132,14 +153,31 @@ def config_to_dict(config_file: str) -> dict:
     Returns:
         dictionary: dictionary of config file
     """
+
+    # Check that config file exists and is a file
+    if not os.path.exists(config_file):
+        raise FileNotFoundError(config_file)
+    if not os.path.isfile(config_file):
+        raise RuntimeError(f"config_file is a directory: '{config_file}'")
+
+    # Define list of config file options that we expect to be paths
+    options_that_are_paths = ["datm_file"]
+
     config = configparser.ConfigParser()
     config.read(config_file)
 
     dictionary = {}
     for section in config.sections():
         dictionary[section] = {}
         for option in config.options(section):
-            dictionary[section][option] = config.get(section, option)
+            value = config.get(section, option)
+
+            # If the option is one that we expect to be a path, ensure it's an absolute path.
```

Comment:
It seems like some of these updates could be moved into the `check_arg_validity` method. I think we could also combine your new `get_abspath_from_config_file` with the other file checking methods that are already here and just use it instead.

Reply:
I moved the config file check to `check_arg_validity()`. I'm not sure what you mean about `get_abspath_from_config_file()`, though?